### PR TITLE
A new article about PCSD

### PR DESCRIPTION
**Pull request by dsocha**

### CODE REVIEW

File: source/2017-07-21-pcsd/index.html.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+![oauth](oauth.png)
+
+## Export data to CSV
+
+To export the data to CSV file you have to execute PCSD.exe application with following command prompt parameters:
+
+* **clientid**: Client ID generated during configuration of oAuth integration.
+* **clientsecret**: Client Secret generated during configuration of oAuth integration.
+* **environment**: Environment's address based on your region, e.g. mypurecloud.ie for EMEA.
```
Comment (dsocha) [inline source/2017-07-21-pcsd/index.html.markdown]: could you pls elaborate this comment

**Comment by PrinceNebulon** [comment]
I can merge this once the remaining feedback has been addressed.

File: source/2017-07-21-pcsd/index.html.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: Accessing the PureCloud data using SQL queries or MS Excel
```
Comment (dsocha) [inline source/2017-07-21-pcsd/index.html.markdown]: Ha ha, it was a team project and we voted for the name together. I will ask my TM what does he think about changing the name for the tool.

Comment (dsocha) [inline source/2017-07-21-pcsd/index.html.markdown]: Hey, after internal discussion we are not going to change the PCSD name for now.

**Pull request merged**